Generator/main.swift

```diff
@@ -89,7 +89,7 @@ func main() {
         valueDict?.removeValue(forKey: "enum")
         deviceList[hardwareKey] = valueDict
     }
-    let dirPath = "../Source/"
+    let dirPath = "../Sources/"
```

Comment:
I think this will break the cocoapod, need to update that file as well. Have to fix that one. Will check it

Reply:
You are totally right, sorry I missed that!

Reply:
It's all right, i fixed it